event/event.gen.go

```diff
@@ -125,12 +145,6 @@ func Close(listener func(*vecty.Event)) *vecty.EventListener {
 	return &vecty.EventListener{Name: "close", Listener: listener}
 }
 
-// The compass used to obtain orientation data is in need of calibration.
-// https://developer.mozilla.org/docs/Web/Events/compassneedscalibration
-func CompassNeedsCalibration(listener func(*vecty.Event)) *vecty.EventListener {
```

Comment:
It looks like it's no longer listed in the table at https://developer.mozilla.org/en-US/docs/Web/Events.

There is still a mention at the sensor events section:

![image](https://cloud.githubusercontent.com/assets/1924134/12861975/23d9f0a6-cc1c-11e5-9171-7fb09191f39e.png)

But the link is 404:

https://developer.mozilla.org/en-US/docs/Web/Events/compassneedscalibration